AzureCalling/External/Calling/CallingContext.swift

```diff
@@ -226,6 +226,27 @@ class CallingContext: NSObject {
         }
     }
 
+    func switchCamera(completionHandler: @escaping (Result<Void, Error>) -> Void) {
+        guard let localVideoStream = localVideoStream else {
+            return
+        }
+
+        let currentCamera = localVideoStream.source
+        let flip: CameraFacing = currentCamera.cameraFacing == .front ? .back : .front
+
+        if let flipCamera = availableCameras[flip] {
+            localVideoStream.switchSource(camera: flipCamera) { error in
+                if error != nil {
+                    print("ERROR: It was not possible to switch camera. \(error!)")
+                    completionHandler(.failure(error!))
+                    return
+                }
+                print("Camera switch successful")
+                completionHandler(.success(()))
```

Comment:
if there is no flipCamera available, we still need to send out a completionHandler(.failure)

Reply:
Good point, we still need to handle all cases even though it's unlikely to happen. `.failure` does not accept optional Error value though.